src/client/css/partials/user-menu.css

```diff
@@ -0,0 +1,120 @@
+.user-menu-wrapper {
+  font: normal 14px/1.5 Inter, Inter-fallback, sans-serif;
+  outline: none;
+  position: relative;
+}
+
+.user-menu-button {
+  border: 2px solid transparent;
+  background: var(--gray-10);
+  cursor: pointer;
+  display: block;
+  border-radius: 50%;
+  height: 42px;
+  overflow: hidden;
+  transition: border-color 150ms ease;
+  width: 42px;
+}
+
+.user-menu-button:hover,
+.user-menu-button:focus {
+  border-color: var(--purple-70);
+}
+
+.user-menu-list > :nth-child(1) {
+  margin-bottom: var(--padding-sm);
+}
+
+.user-menu-popover {
+  background: white;
+  border-radius: 6px;
+  box-shadow: 0px 2px 12px rgba(0, 0, 0, 0.15);
+  list-style: none;
+  margin: var(--padding-md) 0 0 0;
+  min-width: var(--min-width);
+  max-width: var(--mobile-breakpoint);
+  padding: var(--padding-sm) 0;
+  position: absolute;
+  right: 0;
+}
+
+.user-menu-popover::after {
+  background-color: white;
+  border-top-left-radius: 1px;
+  content: '';
+  display: block;
+  height: 14px;
+  position: absolute;
+  right: calc(46px * 0.5);
+  top: 0;
+  transform: translate(50%, -50%) rotate(45deg);
+  width: 14px;
+}
+
+.user-menu-popover.visible {
+  opacity: 1;
+}
+
+.user-menu-header {
+  align-items: flex-start;
+  display: flex;
+  gap: var(--padding-xs);
+  flex-direction: column;
+  padding: var(--padding-md) calc(var(--padding-md) * 1.5);
+  position: relative;
+  text-decoration: none;
+}
+
+.user-menu-header::after {
```

Comment:
nit/fun experiment: I _think_ you could have just used an `<hr>` here instead of pseudo?  It think `<hr>` is as semantic as it gets, and might only require a single line of CSS: `border-image: var(--monitor-gradient) 1;`  What you have is perfectly fine too – only explore `<hr>` if you like fun 😄

Reply:
Cool — just replaced the pseudo element! That must be the first time I’m utilizing `border-image`.